psi4/driver/procrouting/dft/hyb_functionals.py

```diff
@@ -149,7 +149,7 @@
         }
     },
     "dispersion": {
-        "type": "d3bj",
+        "type": "d3bj2b",
```

Comment:
Worth checking if ATM should stay off. Can't recall what the paper recommended.

Reply:
According to eqns 42, 44, 52 in https://psi-k.net/download/highlights/Highlight_144.pdf , ATM is off, on, on for HF-3c, PBEh-3c, B97-3c. But I also realize I don't know if gCP could be covering that.

For the moment, this change is preserving the previous behavior. So there's still the opportunity for a deliberate fix to match the literature later.

Reply:
Agree with preserving the old behavior for now.